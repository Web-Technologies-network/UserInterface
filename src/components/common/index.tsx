export { Card } from './Card/Card';
export { Danger } from './Typography/Danger';
export { Info } from './Typography/info';
export { Success } from './Typography/Success';
export { Warning } from './Typography/Warning';
