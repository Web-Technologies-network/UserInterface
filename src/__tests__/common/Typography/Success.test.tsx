import React from 'react';
import { Success } from '../../../components/common';
import testRenderer from 'react-test-renderer';

describe('Success', () => {
  let container: string | Node | null; // this will hold the main div where our test component is rendered
  // before each test hook aka init
  beforeEach(() => {
    container = document.createElement('div');
    document.body.append(container);
  });
  // tear down
  afterEach(() => {
    document.body.removeChild(container as Node);
    container = null;
  });
  it('should render the same', () => {
    const tree = testRenderer.create(
      <Success>
        <div>test children</div>
      </Success>,
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
