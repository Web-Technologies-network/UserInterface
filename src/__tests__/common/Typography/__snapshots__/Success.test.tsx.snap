// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success should render the same 1`] = `
<div
  className="MuiGrid-root makeStyles-defaultFontStyle-1 makeStyles-successText-9"
>
  <div>
    test children
  </div>
</div>
`;
